### DISCUSSION TITLE
Play in this state is not permitted. lazer

### DISCUSSION THREAD

**Discussion by CufeeEz** [ppy/osu#28531] [Q&A]
Problem details:
trying to access the maps in multiplayer or playlists gives an error [1].
the same error occurs in single player mode, but I can play cards, but nothing counts.
I checked the latest updates [2].
I installed it from the official website again, without deleting the application (like on top of this), this method helped, but as you can see, not for long.
how can this be solved without reinstalling a million times?
Video or screenshot showing the problem:
[1] ![image](https://github.com/ppy/osu/assets/92633278/bb403029-cc22-4317-a0b4-faea98c4d730)
[2] ![image](https://github.com/ppy/osu/assets/92633278/f4477e06-fcb3-45db-bf29-73cb963cd376) translation from Russian to English (You are playing the latest version of OSU! (2024.521.2-lazer))

osu! version: 2024.521.2-lazer

All this happens on the account https://osu.ppy.sh/users/31790595
[logs.zip](https://github.com/user-attachments/files/15904905/logs.zip)

**Comment by Stoppedpuma** [comment]
I can't seem to unzip your logs, can you use the export logs button found here?

![image](https://github.com/ppy/osu/assets/58333920/66f9feb5-c1ec-457e-9c9e-2724a601898f)

**Reply by CufeeEz** [reply]
> It's a RAR archive renamed to zip, try opening it as a RAR

yes, I did this because github does not allow me to upload files in rar format

**Comment by sineplusx** [comment]
Your logs show that you run the game with elevated privileges ("Administrator" on Windows). There is a popup every time you start the game warning you of this fact. Running osu! lazer this way is not supported. If you for some reason can't run the game as a regular user, we can figure out a solution for that instead.

```
2024-06-19 18:42:05 [verbose]: ⚠️ Running osu! as administrator does not improve performance, may break integrations and poses a security risk. Please run the game as a normal user.
```

**Reply by CufeeEz** [reply]
this was done in anticipation that the error would go away
usually the game starts without administrator permission

**Reply by sineplusx** [reply]
So this also happens when you run it without administrator privileges?
You can try removing the file `AuthNative.dll` in `%APPDATA%\osu\` (the default lazer data location)
This is the same issue as in https://github.com/ppy/osu/issues/27290

**Reply by CufeeEz** [reply]
> So this also happens when you run it without administrator privileges? You can try removing the file `AuthNative.dll` in `%APPDATA%\osu\` (the default lazer data location)

I deleted it, the error still remains. In general, I noticed that the game itself allegedly runs on behalf of the administrator

https://github.com/ppy/osu/assets/92633278/f15aa8fd-9455-443a-bb23-16d8f73202ae

**Reply by Stoppedpuma** [reply]
> I noticed that the game itself allegedly runs on behalf of the administrator

Check if you have disabled UAC, this is a really stupidly designed thing and apparently runs everything as administrator if it's disabled.

**Reply by CufeeEz** [reply]
> > I noticed that the game itself allegedly runs on behalf of the administrator
> 
> Check if you have disabled UAC, this is a really stupidly designed thing and apparently runs everything as administrator if it's disabled.

I disabled UAC in every possible way, through the control panel, regedit and gpedit.msc, but the warning still pops up. Despite this, sometimes scores pass, but not always

**Reply by Stoppedpuma** [reply]
You seem to have misunderstood, **do not disable UAC** it holds basic security features and is most likely the reason you're facing this issue.
